### PR TITLE
Solved: New Navbar design and functionality also added dropdown card 🚀 [New Frontpage Issue#2] #296

### PR DESCRIPTION
**Pull request by crocmons**
New Navbar design also added the dropdown card, toggle button 🚀 [New Frontpage Issue#2] #296

[screencast-www.linkedin.com-2023.06.19-15_25_45.webm](https://github.com/anmode/grabtern-frontend/assets/106610322/f115a801-4551-468b-8b35-a1fb232c44de)

### CODE REVIEW

**Comment by crocmons** [comment]
> The build is failing!! Really thanks for this.
> 
> 1. Please remove dark mode and that toggle too. We will work on dark mode later with figma Ui

ok, let me do this

**Comment by crocmons** [comment]
you just need the toggle design right not the functionality?

**Comment by crocmons** [comment]
done please check now @anmode

**Review by anmode — COMMENTED** [review]
cool! @crocmons I see one of outreachy card in the internship section is distorted.. Please could you check that? 

and yes remove that toggle... or you can comment it out we will use afterward when dark mode would be ready

**Comment by anmode** [comment]
I think this new logo would look good: https://www.canva.com/design/DAFlzVM_kfs/FJmsajDd_Kv2PgN0VF5UzQ/edit?utm_content=DAFlzVM_kfs&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton
 what say?

**Comment by glokta1** [comment]
@crocmons Hi, thanks for The PR! Just one thing:

The background color is not white but more off-white/light grey. Please change this

**Review by muhammadjufry — DISMISSED** [review]
Please fix it:
1. The logo is small
2. The actual navbar content was wrapped by a container which had a max-width of around 1200px and a margin auto this version doesn't have a container so the content only had some space between the end layer and the content

Other than that it looks nice 👌

**Comment by crocmons** [comment]
> I think this new logo would look good: https://www.canva.com/design/DAFlzVM_kfs/FJmsajDd_Kv2PgN0VF5UzQ/edit?utm_content=DAFlzVM_kfs&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton what say?

yeah, it looks great..so can I add this new logo on the navbar?

**Comment by crocmons** [comment]
> @crocmons Hi, thanks for The PR! Just one thing:
> 
> As mentioned in the issue, the background color is not white but more off-white/light grey. Please change this.

yeah sure,.. thanks @glokta1

**Comment by anmode** [comment]
> > I think this new logo would look good: https://www.canva.com/design/DAFlzVM_kfs/FJmsajDd_Kv2PgN0VF5UzQ/edit?utm_content=DAFlzVM_kfs&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton what say?
> 
> yeah, it looks great..so can I add this new logo on the navbar and remove the previous logo? @anmode

yess!!

**Comment by crocmons** [comment]
done.. done.. Added a new logo in the header.

**Comment by anmode** [comment]
Some fixes:
1. Logo looking so small
    - In canva I enlarged the pic as you downloaded it with white background
    - I think it is good you can download png ( without background)
2.
Screenshot 2023-06-21 at 9 27 54 PM
This does not look good:

we can have -
Screenshot 2023-06-21 at 9 29 15 PM
In mobile mode in place we can have simple dropdown with option `as user` or `as mentor`


Or if you have any design idea that would be great!!

File: tailwind.config.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  prefix: "tw-",
```
Comment (anmode) [inline tailwind.config.js:3]: Please do not remove it! other parts used this config option

Comment (crocmons) [reply tailwind.config.js:3]: @anmode if I add this to the tailwind configuration file my style code will be deleted. Screenshot (425)

Comment (crocmons) [reply tailwind.config.js:3]: @anmode ok so I have to add tw- in front of tailwind classes

**Comment by crocmons** [comment]
Updated.. Hope you liked!
Screenshot (427)

File: tailwind.config.js (hunk 1 above)
Comment (anmode) [reply tailwind.config.js]: yes yeS!

**Comment by anmode** [comment]
Some fixes and good to go:
1. Remove the search bar I think we would need that ( please just comment in out)
2. If logo size can increased that would be good
3. card distorted in mobile mode
Screenshot 2023-06-22 at 6 28 02 AM
I guess dropdown in mobile mode would be looks better ( sign button will not be button can same as other options but which a down arrow to show that this is drop down) - How's that sounds?

File: components/Header.js [hunk 2]
```diff
@@ -1,32 +1,29 @@
...
+import { RxHamburgerMenu } from "react-icons/rx";
+import DropdownCard from "./LoginDropdown";
+import { AiOutlineSearch } from "react-icons/ai";
+import { SunIcon } from "@heroicons/react/24/solid";
+import logo from "../public/logo.png";
 
-function Header({ navbarBackground }) {
-  // localStorage.setItem("redirectUrl", window.location.href);
-  // const [isLoggedIn, setLoggedIn] = useState(false);
+function Header({ isUserLoggedIn }) {
```
Comment (anmode) [inline components/Header.js:11]: we are using `Authcontext` for managing  auth state So i think we do not need to pass here!

File: components/Header.js [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
-  const [navbarAppear, setNavbarAppear] = useState(false);
-  const [loginOption, setLoginOption] = useState(false);
-  const {
-    isMentorLoggedIn,
-    setIsMentorLoggedIn,
-    isUserLoggedIn,
-    setIsUserLoggedIn,
-  } = useAuth();
-  const userData = JSON.parse(localStorage.getItem("userData"));
-  const mentorData = JSON.parse(localStorage.getItem("mentorData"));
```
Comment (anmode) [inline components/Header.js:21]: please do not rmv this:
1. useAuth manage the auth state
2. We have stored data in that format in local storage so hence we parse too

File: components/Header.js [hunk 4]
```diff
@@ -1,32 +1,29 @@
...
+  const [toggleDropdown, setToggleDropdown] = useState(false);
+  const [searchText, setSearchText] = useState("");
+  const dropdownRef = useRef();
 
   useEffect(() => {
-    if (userData?.user_name) {
-      setIsUserLoggedIn(true);
+    const userName = localStorage.getItem("user_name");
+    if (userName) {
+      setLoggedIn(true);
```
Comment (anmode) [inline components/Header.js:22]: revert back!

File: components/Header.js [hunk 5]
```diff
@@ -1,32 +1,29 @@
...
     }
-
-    if (mentorData?.mentor_name) {
-      setIsMentorLoggedIn(true);
+    const mentorName = localStorage.getItem("mentor_name");
+    if (mentorName) {
+      setMentorLoggedIn(true);
```
Comment (anmode) [inline components/Header.js:26]: same!

File: components/LoginDropdown.js [hunk 6]
```diff
@@ -1,41 +1,30 @@
...
-    setIsMentorLoggedIn,
-    isUserLoggedIn,
-    setIsUserLoggedIn,
-  } = useAuth();
+  const [isLoggedIn, setLoggedIn] = useState(false);
+  const [isMentorLoggedIn, setMentorLoggedIn] = useState(false);
   const dropdownRef = useRef(null);
-  const userData = JSON.parse(localStorage.getItem("userData"));
-  const mentorData = JSON.parse(localStorage.getItem("mentorData"));
-  const router = useRouter();
```
Comment (anmode) [inline components/LoginDropdown.js:17]: same here! do not remove auth, and revert back

File: components/LoginDropdown.js [hunk 7]
```diff
@@ -1,41 +1,30 @@
...
-    } else {
-      setLoginOption(true);
-    }
+    setLoginOption(!loginOption);
   };
 
   const mentorlogout = () => {
     localStorage.clear();
-    setIsMentorLoggedIn(false);
+    setMentorLoggedIn(false);
```
Comment (anmode) [inline components/LoginDropdown.js:18]: revert back!

**Review by anmode — CHANGES_REQUESTED** [review]
leave some comments!

**Comment by crocmons** [comment]
> Some fixes and good to go:
> 
> 1. Remove the search bar I think we would need that ( please just comment in out)
> 2. If logo size can increased that would be good
> 3. card distorted in mobile mode
> 
> Screenshot 2023-06-22 at 6 28 02 AM I guess dropdown in mobile mode would be looks better ( sign button will not be button can same as other options but which a down arrow to show that this is drop down) - How's that sounds?

Ok, that's great,,,so I have to add drop-down arrow icon in the mobile mode instead of button like this?
pre

**Comment by anmode** [comment]
> > Some fixes and good to go:
> > 
> > 1. Remove the search bar I think we would need that ( please just comment in out)
> > 2. If logo size can increased that would be good
> > 3. card distorted in mobile mode
> > 
> > Screenshot 2023-06-22 at 6 28 02 AM I guess dropdown in mobile mode would be looks better ( sign button will not be button can same as other options but which a down arrow to show that this is drop down) - How's that sounds?
> 
> Ok, that's great,,,so I have to add drop-down arrow icon in the mobile mode instead of button like this? pre

yes yes!! signin will be same like home , mentor but it will have a arrow which on click open a dropdown with two options

**Comment by anmode** [comment]
@crocmons if you facing setting this dropdown I have other things in my mind:
on clicking sign up button in mobile and desktop mode ( as there is theme)
- It redirect to a page on one side two mode as user or as mentor ( what's say)? 
and you can do this feature in your follow up pR so this issue would be divided into two level 3 issue.

**Comment by crocmons** [comment]
> @crocmons if you facing setting this dropdown I have other things in my mind: on clicking sign up button in mobile and desktop mode ( as there is the theme so we can have exact same nav as of our theme)
> 
> * It redirect to a page on one side two modes as a user or as a mentor ( what's say)?
>   and you can do this feature in your follow-up pR so this issue would be divided into two level 3 issues.

yeah, I guess that should be better if you suggest me.... do you want to say if a user clicks that button it will redirect a page where users can easily sign up as a user or mentor? ..

**Comment by anmode** [comment]
> > @crocmons if you facing setting this dropdown I have other things in my mind: on clicking sign up button in mobile and desktop mode ( as there is the theme so we can have exact same nav as of our theme)
> > 
> > * It redirect to a page on one side two modes as a user or as a mentor ( what's say)?
> >   and you can do this feature in your follow-up pR so this issue would be divided into two level 3 issues.
> 
> @anmode yeah, I guess that should be better if you suggest me.... Do you want to say if a user clicks that button it will redirect a page where users can easily sign up as a user or a mentor? Like this way?.. ScreenRecorderProject22

yeah yeah! it look cool!

**Comment by anmode** [comment]
`Signin` when anybody clicks so it redirects to the page as you show with two options User or Mentor. It would be cool

**Comment by anmode** [comment]
@crocmons In this PR can you replicate this nav bar exactly?
https://grabtern-new-frontend.vercel.app/ ( in light mode)
Mobile mode too exact same ( As in this theme there is not card so you comment the card out as of now)
Differnce will be Replace SignUp button with Login and remove sign in button


In another follow up pR you can create that UI as you share which wilil open when login button is clicked so we will have slider and form ( as you shared) with Mentor or User

**Comment by crocmons** [comment]
> @crocmons In this PR can you replicate this nav bar exactly? https://grabtern-new-frontend.vercel.app/ ( in light mode) Mobile mode too exact the same ( As in this theme there is no card so you comment the card out as of now) The difference will be to Replace the SignUp button with Login and remove sign in button
> 
> In another follow-up pR you can create that UI as you share which will open when the login button is clicked so we will have a slider and form ( as you shared) with the Mentor or User

yeah sure @anmode I can replace this with new navbar..

**Comment by anmode** [comment]
Cool! Thanks

File: components/Header.js (hunk 2 above)
Comment (crocmons) [reply components/Header.js:11]: done

File: components/Header.js (hunk 4 above)
Comment (crocmons) [reply components/Header.js:22]: done

File: components/LoginDropdown.js (hunk 7 above)
Comment (crocmons) [reply components/LoginDropdown.js:18]: done

**Comment by anmode** [comment]
@crocmons please check the video I posted in the above comment... the nav should exactly behave and you take code of that too I provided the repo.

**Comment by crocmons** [comment]
> @crocmons please check the video I posted in the above comment... the nav should exactly behave and you take code of that too I provided the repo.

Ok, but I think the Navbar looks same here as you showed above in that video there is just a difference between those logos... Can you tell me what should I exactly add to this Navbar?

**Comment by anmode** [comment]
1. Remove card and for mobile mode when signup button is clicked there should be a dropdown as in theme but now card is coming.
2. On scrolling nav color changes to white...
Just try to see the behaviour of nav once in new theme in desktop and mobile mode please

**Comment by anmode** [comment]
https://grabtern-new-frontend.vercel.app/

As i can see nav bar color is transparent but on scrolling white background comes look!
actually we want exact same UI in mobile and desktop as in theme

**Comment by crocmons** [comment]
> https://grabtern-new-frontend.vercel.app/
> 
> As i can see nav bar color is transparent but on scrolling white background comes look! actually we want exact same UI in mobile and desktop as in theme

ok, thanks, now I get the point..

**Comment by anmode** [comment]
> ok, thanks, now I get the point & I will do it as you mentioned above.@anmode

That's cool! If any futhure detailing required please let me know...
Some times duplicating UI is tough but here's I provided code too. Hope it helps you in designing.

**Comment by crocmons** [comment]
> > ok, thanks, now I get the point & I will do it as you mentioned above.@anmode
> 
> That's cool! If any futhure detailing required please let me know... Some times duplicating UI is tough but here's I provided code too. Hope it helps you in designing.

done..please check it @anmode..

**Comment by anmode** [comment]
For desktop mode:
1. See in theme Nav bar is transparent (Actually there is PR which will change hero section and we will have grey color that you do not have to put grey color in nav)
2. The UI of signup button in Theme looks cool! coloured shadow effect

Screenshot 2023-06-24 at 7 58 55 AM

For mobile mode:
1. In theme we have drop-down same is required
2. UI needs to be improved

**Comment by anmode** [comment]
@crocmons see this : https://github.com/anmode/grabtern-frontend/pull/328
your PR will be merged first then this pr

**Comment by anmode** [comment]
Actually exact same look is to be required:

https://github.com/anmode/grabtern-frontend/assets/88332977/cfa5bfba-9720-44bf-8597-5b479f96e8f0

**Comment by anmode** [comment]
If there is any miscommunication or you aren't getting any point we can have Gmeet. Please let me know,
Thanks,
Actually It could be very simple you just have to copy code and make it work here!

**Comment by crocmons** [comment]
> For desktop mode:
> 
> 1. See in theme Nav bar is transparent (Actually there is PR which will change hero section and we will have grey color that you do not have to put grey color in nav)
> 2. The UI of signup button in Theme looks cool! coloured shadow effect
> 
> Screenshot 2023-06-24 at 7 58 55 AM
> For mobile mode:
> 
> 1. In theme we have drop-down same is required
> 2. UI needs to be improved

@anmode ok, that's fine no problem ..After he creates the hero section please let me know then I will do the rest of the work..because if I add the transparent now ( tw-bg-muted-100) on this current navbar it looks weird so I think when he completed his pr then I will do it then it's better for me to work on it..

**Comment by anmode** [comment]
> > For desktop mode:
> > 
> > 1. See in theme Nav bar is transparent (Actually there is PR which will change hero section and we will have grey color that you do not have to put grey color in nav)
> > 2. The UI of signup button in Theme looks cool! coloured shadow effect
> > 
> > Screenshot 2023-06-24 at 7 58 55 AM
> > For mobile mode:
> > 
> > 1. In theme we have drop-down same is required
> > 2. UI needs to be improved
> 
> @anmode ok, that's fine no problem ..After he creates the hero section please let me know then I will do the rest of the work..because if I add the transparent now ( tw-bg-muted-100) on this current navbar it looks weird so I think when he completed his pr then I will do it so that it's better for me to work on it..

Okay..for now if you feel like.
Can optimize mobile.
As in theme - there is dropdown ( hamburger menu)
Could you replicate it?

**Comment by crocmons** [comment]
ok

**Comment by crocmons** [comment]
Hey, Hi @anmode hope you're well..It's been a long time since you're not giving me any response about further updates so I just completed my pr and now it looks like as you mentioned above.
 
[screencast-localhost_3000-2023.07.02-03_23_58.webm](https://github.com/anmode/grabtern-frontend/assets/106610322/91d7abf3-bc66-4902-aae0-4e60c1a6f77b)

**Comment by crocmons** [comment]
Also, the resolve conflicts button was disabled ...So, I can't resolve it...@anmode

**Comment by anmode** [comment]
Sorry for late response,
Wow it is good to merge please resolve the conflicts and also some color changes are needed. please see the theme which i provided ( light mode)

**Comment by anmode** [comment]
> leave some comments!

please have a look to these comments!

**Comment by crocmons** [comment]
> @crocmons Sorry for late response, Actually that home page PR is still pending.. That's why I did not say here.... Wow it is good to merge please resolve the conflicts and also some color changes are needed. please see the theme which i provided ( light mode)

Yeah, that's fine ..Some color change..Can you explain to me which color you need for this navbar..? So that I can work on it.. @anmode

**Comment by anmode** [comment]
> > @crocmons Sorry for late response, Actually that home page PR is still pending.. That's why I did not say here.... Wow it is good to merge please resolve the conflicts and also some color changes are needed. please see the theme which i provided ( light mode)
> 
> Yeah, that's fine ..Some color change..Can you explain to me which color you need for this navbar..? So that I can work on it.. @anmode

same as in theme! light mode

**Comment by crocmons** [comment]
> > > @crocmons Sorry for late response, Actually that home page PR is still pending.. That's why I did not say here.... Wow it is good to merge please resolve the conflicts and also some color changes are needed. please see the theme which i provided ( light mode)
> > 
> > 
> > Yeah, that's fine ..Some color change..Can you explain to me which color you need for this navbar..? So that I can work on it.. @anmode
> 
> same as in theme! see light mode

Sorry, but I didn't get it.. https://github.com/anmode/grabtern-frontend/pull/372#issuecomment-1604576465

**Comment by anmode** [comment]
> > > > @crocmons Sorry for late response, Actually that home page PR is still pending.. That's why I did not say here.... Wow it is good to merge please resolve the conflicts and also some color changes are needed. please see the theme which i provided ( light mode)
> > > 
> > > 
> > > Yeah, that's fine ..Some color change..Can you explain to me which color you need for this navbar..? So that I can work on it.. @anmode
> > 
> > 
> > same as in theme! see light mode
> 
> Sorry, but I didn't get it.. [#372 (comment)](https://github.com/anmode/grabtern-frontend/pull/372#issuecomment-1604576465)

Oh sorry! It was my fault. it comes in hero section PR. Not nav bar work.

**Comment by anmode** [comment]
One thing @crocmons here what I suggest.
Keep this PR:
1. You can replicate Hero section , I will merge that PR.
2. Then nav bar
and nav bar code you can take from this PR. If after hero section you wanna continue navbar in this PR that also would be cool

**Comment by anmode** [comment]
> > One thing @crocmons here what I suggest. Keep this PR:
> > 
> > 1. You can replicate Hero section , I will merge that PR.
> > 2. Then nav bar
> >    and nav bar code you can take from this PR. If after hero section you wanna continue navbar in this PR that also would be cool
> > 
> > are you up for this proposal please let me know?, thanks
> 
> Actually my task is only navbar design & functionality that you mentioned in #296 ..@anmode

Ah, I know that's why I wrote, this is just a proposal.
If you agree then ok. I make it fair too as you can have two level 3 prs one is hero section and other nav which is somway link.

**Comment by crocmons** [comment]
> > > One thing @crocmons here what I suggest. Keep this PR:
> > > 
> > > 1. You can replicate Hero section , I will merge that PR.
> > > 2. Then nav bar
> > >    and nav bar code you can take from this PR. If after hero section you wanna continue navbar in this PR that also would be cool
> > > 
> > > are you up for this proposal please let me know?, thanks
> > 
> > 
> > Actually my task is only navbar design & functionality that you mentioned in #296 ..@anmode
> 
> Ah, I know that's why I wrote, this is just a proposal. If you agree then ok. I make it fair too as you can have two level 3 prs one is hero section and other nav which is somway link.

ok, Let me try it...Can you give me the issue number on that hero section?

**Comment by crocmons** [comment]
> Please let me know if you are up for this? If not I will review the pR carefully. Then and will complete this pR asap

Firstly Can you merge this #296 pr? So that I can work on the hero section easily with this new navbar...

**Comment by anmode** [comment]
> > Please let me know if you are up for this? If not I will review the pR carefully. Then and will complete this pR asap
> 
> Firstly Can you merge this #296 pr? So that I can work on the hero section easily with this new navbar because there is a color issue as you mentioned above...

Ok some improvement is needed here, 
1. Sign Up button UI improved with the theme 
2. Please use our theme color or you can use button from component/Ui
3. I see you give shadow to nav, I think it is good to remove it

File: components/basic/LoginDropdown.js [hunk 8]
```diff
@@ -12,30 +15,23 @@ const DropdownCard = () => {
     setIsUserLoggedIn,
   } = useAuth();
   const dropdownRef = useRef(null);
-  const userData = JSON.parse(localStorage.getItem("userData"));
-  const mentorData = JSON.parse(localStorage.getItem("mentorData"));
-  const router = useRouter();
```
Comment (anmode) [inline components/basic/LoginDropdown.js:17]: why you removed this?

File: components/basic/LoginDropdown.js [hunk 9]
```diff
@@ -45,102 +41,85 @@ const DropdownCard = () => {
...
+            style={{
+              width: "30px",
+              height: "auto",
+              marginRight: "10px",
+              borderRadius: "50%",
+              display: "inline",
+            }}
+            src={
+              localStorage.getItem("user_picture") ||
+              localStorage.getItem("mentor_picture") ||
```
Comment (anmode) [inline components/basic/LoginDropdown.js:64]: Actually we are using in a userdata and mentordata json.
See my previous comment

**Review by anmode — CHANGES_REQUESTED** [review]
I'll give a final review when UI will be fixed

File: components/basic/LoginDropdown.js (hunk 9 above)
Comment (crocmons) [reply components/basic/LoginDropdown.js:64]: Resolved

File: components/basic/LoginDropdown.js (hunk 8 above)
Comment (crocmons) [reply components/basic/LoginDropdown.js]: ok at that time I guess it is not needed ...but now I add this code pls check...@anmode

File: components/basic/LoginDropdown.js (hunk 9 above)
Comment (crocmons) [reply components/basic/LoginDropdown.js]: I added the prev code as you mentioned..

**Comment by crocmons** [comment]
Done check it now -

Screenshot (487)

**Comment by anmode** [comment]
Screenshot 2023-07-07 at 8 11 06 PM
In preview.....

File: components/basic/LoginDropdown.js (hunk 8 above)
Comment (anmode) [reply components/basic/LoginDropdown.js]: ok cool!

**Comment by anmode** [comment]
I checked on local too, there is some issue. Same that dark blue color is there

**Comment by crocmons** [comment]
> I checked on local too, there is some issue. Same that dark blue color is there

 button color ? or navbar color?

**Comment by anmode** [comment]
https://github.com/anmode/grabtern-frontend/assets/88332977/d7adcd67-9ae6-4f72-be1c-ccc0798fb97b

have a look! Please fix it

**Comment by anmode** [comment]
> > I checked on local too, there is some issue. Same that dark blue color is there
> 
> @anmode button color ? or navbar color?

nav bg color!

**Comment by crocmons** [comment]
ohh you're talking about the color change in the dark mode...yeah check it now

**Comment by anmode** [comment]
One more thing as you used blue/violet  color but our theme color is #845ec2.
Please use this or we in component folder we have button themes you can pick any

**Comment by anmode** [comment]
R u available for quick Gmeet?

**Comment by crocmons** [comment]
is this the color you want?

Screenshot (2)

**Comment by crocmons** [comment]
> yes!, cross sign color should also be changed to theme color

cool..let me give it on the toggle dropdown

**Comment by crocmons** [comment]
pls check it now

Screenshot (488)

**Comment by crocmons** [comment]
is this ok for you?

**Comment by anmode** [comment]
Nice, Last could you show me after login behavior too? 
Some changes:
1. Make button hover effect as give shadow and on hover that shadow disappear ( button having this behaviour already present as component in codebase)
On you local visit /components

**Review by anmode — APPROVED** [review]
Cool, LGTM.
The last thing is functionality. and after login behavior of Nav

**Comment by crocmons** [comment]
> Cool, LGTM. The last thing is functionality. and after login behavior of Nav
@anmode you told me that in this pr just focus on the navbar because you create separate dropdown pr for user & mentor https://github.com/anmode/grabtern-frontend/pull/372#issuecomment-1603089506

**Comment by anmode** [comment]
Yeah yeah! But in this issue after login, how will you display profile image and how it look also comes in.

**Comment by crocmons** [comment]
> Yeah yeah! But in this issue after login, how will you display the profile image and how it looks in the nav bar also comes in.

we can add right now only user login for this @anmode ?

**Comment by crocmons** [comment]
> Yeah yeah! But in this issue after login, how will you display the profile image and how it looks in the nav bar also comes in.

how could I know ..you have to create a separate page for this which is redirected to the user..

**Comment by anmode** [comment]
> > Yeah yeah! But in this issue after login, how will you display the profile image and how it looks in the nav bar also comes in.
> 
> how could I know ..you have to create a separate page for this which is redirected to the user..then after login it will appear on the navbar

It is simple for now you can do isUserLoggedIn = true :)

**Comment by anmode** [comment]
Chlo no worries, I'll see it then, 
btw please I request you to continue that toggle that I discussed before.
Again this is a request.

**Comment by anmode** [comment]
Congrats @crocmons ...

**Comment by crocmons** [comment]
> > > Yeah yeah! But in this issue after login, how will you display the profile image and how it looks in the nav bar also comes in.
> > 
> > 
> > how could I know ..you have to create a separate page for this which is redirected to the user..then after login it will appear on the navbar
> 
> It is simple for now you can set `isUserLoggedIn = true` :) and as `userdata = null`, so it will take default profile pic



> > > Yeah yeah! But in this issue after login, how will you display the profile image and how it looks in the nav bar also comes in.
> > 
> > 
> > how could I know ..you have to create a separate page for this which is redirected to the user..then after login it will appear on the navbar
> 
> It is simple for now you can set `isUserLoggedIn = true` :) and as `userdata = null`, so it will take default profile pic

ohh yeap for now I just set it as the default behavior in the login option...thanks @anmode ..finally after a long decade my pr is just merged..😅😂

**Comment by crocmons** [comment]
Should I start working on the hero section or on the user/mentor login page

**Comment by anmode** [comment]
> > > > Yeah yeah! But in this issue after login, how will you display the profile image and how it looks in the nav bar also comes in.
> > > 
> > > 
> > > how could I know ..you have to create a separate page for this which is redirected to the user..then after login it will appear on the navbar
> > 
> > 
> > It is simple for now you can set `isUserLoggedIn = true` :) and as `userdata = null`, so it will take default profile pic
> 
> > > > Yeah yeah! But in this issue after login, how will you display the profile image and how it looks in the nav bar also comes in.
> > > 
> > > 
> > > how could I know ..you have to create a separate page for this which is redirected to the user..then after login it will appear on the navbar
> > 
> > 
> > It is simple for now you can set `isUserLoggedIn = true` :) and as `userdata = null`, so it will take default profile pic
> 
> ohh yeap for now I just set it as the default behavior in the login option...thanks @anmode ..finally after a long decade my pr is just merged..😅😂

yes It grows very long... my bad. Actually that HERO Section :(

**Comment by anmode** [comment]
> Should I start working on the hero section or on the user/mentor login page..Also giving me the issue numbers

ohh sure.... could you create issue too? Your issue count also increased.

**Pull request merged**